Libraries/WebSocket/RCTWebSocket.xcodeproj/project.pbxproj

```diff
@@ -556,6 +556,7 @@
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
```

Comment:
@HeyImChris and @tom-un this is a good reminder that we really need to push these projects to use xcconfigs and ideally bring over some form of our infra that fails builds when settings are inlined in the pbxproj (and obviously push that back to fb). Let's make sure we get that explicitly scheduled soon.

Reply:
The next release of react-native which we’re merging ditches the xcodeproj’s altogether in favor of Cocoa Pod generated xcodeproj’s.   We’ll have to make sure it can leverage shared deployment targets if it doesn’t already.

Reply:
Deployment target should set in the podspec- no more xcodeproj customizations. It looks like people have added post_install hooks that can do this but I haven't tried myself